elixir/apps/web/lib/web/components/core_components.ex

```diff
@@ -1210,6 +1210,14 @@ defmodule Web.CoreComponents do
     for {^field, {msg, opts}} <- errors, do: translate_error({msg, opts})
   end
 
+  def translate_errors(errors, field) when is_map(errors) do
```

Comment:
You can just add `or is_map(errors)` to the clause above without copying the code

Reply:
Appreciate the heads up on this one.  Makes total sense after seeing your comment.  I just didn't realize you could have multiple conditions for a single `when`.  This has been updated.